docs/en/reference/sql/dql/JOIN_CLAUSE.md

```diff
@@ -18,51 +19,219 @@ JoinType ::= 'LAST'
-| JOIN Clause    | Only LAST JOIN is supported | Indicates that the data source multiple tables JOIN. OpenMLDB currently only supports LAST JOIN. During Online Serving, you need to follow [The usage specification of LAST JOIN under Online Serving](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.md#online-serving usage specification of last-join) |
+| `SELECT` Statement Elements                                | Offline Mode | Online Preview Mode | Online Request Mode | Note                                                                                                                                                                                                                                                                                                                                                       |
+|:-----------------------------------------------------------|--------------|---------------------|---------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| JOIN Clause                | **``✓``**    | **``✓``**           | **``✓``**           | The Join clause indicates that the data source comes from multiple joined tables. OpenMLDB currently only supports LAST JOIN. During Online Serving, please follow [the specification of LAST JOIN under Online Serving](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.md#online-servinglast-join)                                                      |
 
-### LAST JOIN without ORDER BY
 
-#### Example: **LAST JOIN Unsorted Concatenation**
 
-```sql
--- desc: simple spelling query without ORDER BY
 
-SELECT t1.col1 as t1_col1, t2.col1 as t2_col2 from t1 LAST JOIN t2 ON t1.col1 = t2.col1
-```
+### LAST JOIN without ORDER BY
 
-When `LAST JOIN` is spliced ​​without sorting, the first hit data row is spliced
+#### Example1: Unsorted LAST JOIN
+
+The unsorted `LAST JOIN` will concat every row of the left table with the last matching row of the right table.  
 
 ![Figure 7: last join without order](../dql/images/last_join_without_order.png)
 
 
-Take the second row of the left table as an example, the right table that meets the conditions is unordered, there are 2 hit conditions, select the last one `5, b, 2020-05-20 10:11:12`
+Take the second row of the left table as an example. The right table is unordered, and there are 2 matching rows. The last one `5, b, 2020-05-20 10:11:12` will be joined with the second row of the left.  
 
 ![Figure 8: last join without order result](../dql/images/last_join_without_order2.png)
 
 The final result is shown in the figure above.
 
+##### SQL Example using OpenMLDB CLI
```

Comment:
I think no need to make any examples as a separate section
=>

**Example of LAST JOIN without ORDER_BY**